INSTALL/lyxblogger.py

```diff
@@ -0,0 +1,919 @@
+#! /usr/bin/env python
```

Comment:
When I run: python setup.py install, it said:

    running install
running build
running build_py
file lyxblogger.py (for module lyxblogger) not found

So I put lyxblogger.py to INSTALL directory, any problem?